src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAsyncEHTests.cs

```diff
@@ -2104,5 +2104,43 @@ static void Main()
 ";
             CompileAndVerify(source, expected);
         }
+
+        [Fact, WorkItem(67091, "https://github.com/dotnet/roslyn/issues/67091")]
+        public void NestedCatch_DuplicateLocal()
```

Comment:
Can we add a test with for a similar scenario in VB?

Reply:
Seems impossible, I'm getting:
<samp>error BC36943: 'Await' cannot be used inside a 'Catch' statement, a 'Finally' statement, or a 'SyncLock' statement.</samp>